session/bootstrap.go

```diff
@@ -44,23 +44,23 @@ import (
 
 const (
 	// CreateUserTable is the SQL statement creates User table in system db.
-	CreateUserTable = `CREATE TABLE if not exists mysql.user (
-		Host				CHAR(64),
-		User				CHAR(32),
+	CreateUserTable = `CREATE TABLE IF NOT EXISTS mysql.user (
+		Host					CHAR(64),
+		User					CHAR(32),
 		authentication_string	TEXT,
-		Select_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Insert_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Update_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Delete_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Create_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Drop_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Select_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Insert_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Update_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Delete_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Create_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Drop_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Process_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Grant_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Grant_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
 		References_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Alter_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Alter_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
 		Show_db_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Super_priv			ENUM('N','Y') NOT NULL DEFAULT 'N',
-		Create_tmp_table_priv		ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Super_priv				ENUM('N','Y') NOT NULL DEFAULT 'N',
+		Create_tmp_table_priv	ENUM('N','Y') NOT NULL DEFAULT 'N',
```

Comment:
Please make these align, also the other SQLs in this file

Reply:
Sorry, it's already done.

Reply:
> Sorry, it's already done.
merge failed.